app/controllers/coordinator_controller.rb

```diff
@@ -2,9 +2,23 @@ class CoordinatorController < ApplicationController
   before_action :auth
 
   def deliver
+    current_deliveries = Transaction.includes(
+        { recipient: [:recipient_profile] }, :donation).where(delivered_at: nil)
+    gon.deliveries = current_deliveries.as_json(
+        {include: [{recipient: {include: [:recipient_profile]}}, :donation]})
```

Comment:
Redundant curly braces around a hash parameter.<br>Indent the first parameter one step more than the previous line.<br>Space inside { missing.<br>Space inside } missing.